pcsx2/x86/microVU_Compile.inl

```diff
@@ -207,7 +207,7 @@ __ri void branchWarning(mV) {
 	incPC(-2);
 	if (mVUup.eBit && mVUbranch) {
 		incPC(2);
-		Console.Error("microVU%d Warning: Branch in E-bit delay slot! [%04x]", mVU.index, xPC);
+		Console.WriteLn(Color_Orange, "microVU%d Warning: Branch in E-bit delay slot! [%04x]", mVU.index, xPC);
```

Comment:
If the user needs to know about it but it's not of major concern then I think `Console.Warning` should be used.
If the user doesn't need to know about it then I think `DevCon.Warning` should be used.